models/classes/CategoryService.php

```diff
@@ -132,27 +132,33 @@ public static function sanitizeCategoryName($value)
      *
      * @param RdfClass $class the $class
      *
-     * @return RdfProperties[] the list of eligible properties
+     * @return RdfProperty[] the list of eligible properties
      */
     public function getElligibleProperties(RdfClass $class)
     {
         $properties = $this->getItemService()->getClazzProperties($class, new RdfClass(self::ITEM_CLASS_URI));
+
         return array_filter(
             $properties,
-            function ($property) {
-                if (in_array($property->getUri(), self::$excludedPropUris)) {
+            function (RdfProperty $property) {
+                if (
+                    in_array($property->getUri(), self::$excludedPropUris, true)
+                    || !$this->doesExposeCategory($property)
+                ) {
                     return false;
                 }
+
                 $widget = $property->getWidget();
-                return !is_null($widget) && in_array($widget->getUri(), self::$supportedWidgetUris);
+
+                return null !== $widget && in_array($widget->getUri(), self::$supportedWidgetUris, true);
```

Comment:
out of curiosity why changing from `!is_null` to `null !===` ?

Reply:
Just a force of habit. Cannot help myself when I see a function call, which can be replaced with an expression.

Reply:
The expression also is a faster than function call. :)